source/tegraexplorer/common/structs.c

```diff
@@ -91,12 +91,13 @@ menu_entry mmcmenu_filemenu[] = {
     {"Part:", COLOR_ORANGE, {ISSKIP | ISMENU}},
     {NULL, COLOR_VIOLET, {ISSKIP | ISMENU}},
     {"\nBack", COLOR_WHITE, {ISMENU}},
-    {"Dump to SD", COLOR_YELLOW, {ISMENU}}
+    {"Dump to SD", COLOR_YELLOW, {ISMENU}},
+    {"No actions possible.", COLOR_RED, {ISSKIP | ISMENU}}
```

Comment:
i don't like the "no actions possible" thing, there are already menus in TE that do not show anything if there is nothing to do